examples/quarkus-cheat-sheet/core.adoc

```diff
@@ -649,136 +649,105 @@ quarkus.log.category."com.lordofthejars".level=DEBUG
 
 Prefix is `quarkus.log`.
 
-|===
-| Property | Default | Description
-a|`category."<category-name>".level`
-a|`INFO`
-a|Minimum level category.
+`category."<category-name>".level`::
+Minimum level category (default: `INFO`)
 
-a|`level`
-a|`INFO`
-a|Default minimum level.
+`level`::
+Default minimum level (default: `INFO`)
 
-a|`console.enable`
-a|`true`
-a|Console logging enabled.
+`console.enabled`::
+Console logging enabled (default: `true`)
 
-a|`console.format`
-a|`%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c{3.}] (%t) %s%e%n`
-a|Format pattern to use for logging.
+`console.format`::
+Format pattern to use for logging. Default value: +
+`%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c{3.}] (%t) %s%e%n`
 
-a|`console.level`
-a|`INFO`
-a|Minimum log level.
+`console.level`::
+Minimum log level (default: `INFO`)
 
-a|`console.color`
-a|`INFO`
-a|Allow color rendering.
+`console.color`::
+// QUESTION: default value should be a boolean right? not INFO?
+Allow color rendering (default: `INFO`)
```

Comment:
The default value should be a Boolean right? not `INFO`?

Reply:
Yes you are right. I am going to fix it in the coming release no worries :)